spring-cloud-starter-stream-common-cdc-debezium/spring-cloud-starter-stream-common-cdc-debezium-core/src/main/java/org/springframework/cloud/stream/app/cdc/common/core/CdcTombstoneConfiguration.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2019 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.cdc.common.core;
+
+import java.util.Collections;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.cloud.stream.annotation.StreamMessageConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.support.KafkaNull;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.converter.AbstractMessageConverter;
+import org.springframework.messaging.converter.MessageConverter;
+
+/**
+ * @author Christian Tzolov
+ */
+@Configuration
+public class CdcTombstoneConfiguration {
```

Comment:
As I said in the other comment, it doesn't look like we need this configuration at all...

Reply:
check my comment above ;)